internal/test/psrpc_test.go

```diff
@@ -159,38 +159,28 @@ type MyService struct {
 	counts map[string]int
 }
 
-func (s *MyService) NormalRPC(ctx context.Context, req *my_service.MyRequest) (*my_service.MyResponse, error) {
-	s.Lock()
```

Comment:
Any motivation for removing the lock here?

Reply:
I'd rather have it panic, if that lock is ever needed something has gone extremely wrong

Reply:
Without a memory barrier, the risk would rather be that an event doesn't get counted. Since it's in a unit test, it's very unlikely to happen indeed though, since updates should not come at the same time.

Reply:
Or crash indeed if the map gets messed up, but same remark on likelihood.